packages/react-admin/src/mui/form/SimpleForm.js

```diff
@@ -4,14 +4,20 @@ import { reduxForm } from 'redux-form';
 import { connect } from 'react-redux';
 import compose from 'recompose/compose';
 import { withStyles } from 'material-ui/styles';
+import withWidth from 'material-ui/utils/withWidth';
 import classnames from 'classnames';
 
 import getDefaultValues from './getDefaultValues';
 import FormInput from './FormInput';
 import Toolbar from './Toolbar';
 
 const styles = {
-    form: { padding: '0 1em 1em 1em' },
+    form: {
```

Comment:
no, the right way is to use `theme.breakpoints`, as in `mui/layout/Layouts.js`. No need to decorate with `withWidth`